frontend/src/components/panels/Document.vue

```diff
@@ -245,7 +246,7 @@ import OptionalInput from "@/components/widgets/inputs/OptionalInput.vue";
 import ToolOptions from "@/components/widgets/options/ToolOptions.vue";
 import { SectionsOfMenuListEntries } from "@/components/widgets/floating-menus/MenuList.vue";
 
-const wasm = import("@/../wasm/pkg");
+const wasm = import("@/../wasm/pkg").then(panicProxy);
```

Comment:
A comment here could be useful. This is cool, though!

Also, is there a way to avoid duplicating this between files? For example, could we make a new file that just exports `import("@/../wasm/pkg").then(panicProxy)`?

Reply:
I'll comment the `panicProxy` function in `panic.ts`.

Unfortunately I cannot find a way to re-export a dynamic module and then dynamically import it, so this was the cleanest approach I could make work.